examples/react-native/storybook/stories/Button.stories.tsx

```diff
@@ -4,8 +4,14 @@ import { action } from '@storybook/addon-actions';
 import { text } from '@storybook/addon-knobs';
 import { storiesOf } from '@storybook/react-native';
 import { Button } from '@aws-amplify/ui-react-native/dist/primitives';
+import { Screen } from '../ui';
```

Comment:
The `Screen` component is simply a full screen `SafeAreaView` component that centers content on the screen.